semperfiwebdesign/all-in-one-seo-pack
# Fix 404 issue for addl-sitemap file

Issue #2794

## Proposed changes

Fixes an issue where the addl-sitemap file still shows up on the root sitemap page when Sitemap Indexes are enabled and when a static homepage is set, but no static posts page.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Checklist


## Testing instructions

Make sure that the addl-sitemap file/index does not show up on the root sitemap page when a static homepage is set, but no static posts page.
